modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorImpl.java

```diff
@@ -149,6 +151,24 @@ protected void validateDDMFormFieldValidationExpression(
 		}
 	}
 
+	private void _findDuplicatedDDMFormFieldReferences(
+		List<DDMFormField> ddmFormFields, Set<String> ddmFormFieldReferences,
+		Set<String> duplicatedDDMFormFieldReferences) {
+
+		for (DDMFormField ddmFormField : ddmFormFields) {
+			if (!ddmFormFieldReferences.add(
+					StringUtil.toLowerCase(ddmFormField.getFieldReference()))) {
```

Comment:
@devpedrotavares why did you add the `StringUtil.toLowerCase`? Shouldn't it be case-sensitive?  Could you leave the same behavior as when we validate the name?

Reply:
I made it case-insensitive, because there is already a validation being done in the front end (although it still allows the submission of the structure) and it's case-insensitive. 

Also, regarding the field names, I noticed that, in addition to being validated in `_validateDDMFormFieldNames`, they are being validated again in `_validateDDMFormFields`. This second one, however, does some extra checks and it is case-insensitive. 
I wonder if this should be refactored in order to be done in just one place.

Reply:
Got it, @devpedrotavares,

> I wonder if this should be refactored in order to be done in just one place.
No need for now.

One more question, will this validation work if we have two fields: "FieldReference" and "fieldReference"? 
The exception should be thrown in this case, right?

Reply:
Yes, the exception would be thrown in this case and the structure wouldn't be created.

Reply:
@devpedrotavares Can you change the test to test this case as well? Instead of `Reference1` you can change to `fieldReference1` and `FieldReference1`? Thanks!

Reply:
Done! I also changed the `Reference`'s to `FieldReference`s in `testDuplicateFieldName` for consistency.